plugins/by-name/nvim-ufo/default.nix

```diff
@@ -130,4 +130,15 @@ lib.nixvim.plugins.mkNeovimPlugin {
         end
       '';
   };
+
+  extraConfig = cfg: {
```

Comment:
Nit:

```suggestion
  extraConfig = {
```

No need for the `cfg` param if it's unused.

Reply:
Although it will be used if we're about to add a `mkIf`

Reply:
Now `cfg` is used with the added option as suggested